src/misc/stringToNumber.ts

```diff
@@ -0,0 +1,9 @@
+/** Returns a numeric representation of an input string.
```

Comment:
@brendandagys This description should start on the next line to maintain consistency with the rest of the project.

Reply:
I have replaced the JSDoc with the snippet from your other comment, and now the description starts on the next line.